examples/facedetection/main.go

```diff
@@ -77,7 +76,7 @@ func main() {
 
 	mediaStream, err := mediadevices.GetUserMedia(mediadevices.MediaStreamConstraints{
 		Video: func(c *mediadevices.MediaTrackConstraints) {
-			c.FrameFormat = prop.FrameFormatExact(frame.FormatI420)
+			//c.FrameFormat = prop.FrameFormatExact(frame.FormatYUY2)
```

Comment:
The FrameFormat needs to be exact since it's used as an optimization to get the Luminance pixels without making a conversion in the face detection, https://github.com/pion/mediadevices/blob/43ac9b56ff290188e9c63b1a039301731085495e/examples/facedetection/main.go#L100. 

An alternative would be `prop.FrameFormatOneOf{frame.FormatYUY2, frame.FormatI420}`